libs/hdf-converters/test/mappers/reverse/checklist_reverse_mapper.spec.ts

```diff
@@ -1,167 +1,157 @@
       'utf-8'
     );
     const converted = mapper.toCkl();
-
     expect(converted).toEqual(replaceCKLVersion(expected));
   });
 });
 
 describe('Small RHEL 7 with severity and severity override tags', () => {
   it('can be successfully converted from HDF to Checklist', () => {
-    const mapper = new ChecklistResults(
-      JSON.parse(
-        fs.readFileSync(
-          'sample_jsons/checklist_mapper/sample_input_report/RHEL7_overrides_hdf.json',
-          {
-            encoding: 'utf-8'
-          }
-        )
-      )
+    const hdfData = loadJsonFile(
+      'sample_jsons/checklist_mapper/sample_input_report/RHEL7_overrides_hdf.json'
     );
-
-    // fs.writeFileSync(
-    //   'sample_jsons/checklist_mapper/converted-rhel7_overrides.ckl',
-    //   mapper.toCkl()
-    // );
-
+    const mapper = new ChecklistResults(hdfData);
     const expected = fs.readFileSync(
       'sample_jsons/checklist_mapper/converted-rhel7_overrides.ckl',
       'utf-8'
     );
     const converted = mapper.toCkl();
-
     expect(converted).toEqual(replaceCKLVersion(expected));
   });
 });
 
 describe('hdf_profile_with_invalid_metadata', () => {
   it('Throws InvalidChecklistFormatException when trying to convert to checklist with invalid metadata', () => {
-    // ensures that checklist metadata is being validated
-    const fileContents = JSON.parse(
-      fs.readFileSync(
-        'sample_jsons/checklist_mapper/sample_input_report/invalid_metadata.json',
-        {encoding: 'utf-8'}
-      )
+    const fileContents = loadJsonFile(
+      'sample_jsons/checklist_mapper/sample_input_report/invalid_metadata.json'
     );
     expect(() => new ChecklistResults(fileContents)).toThrowError(
       InvalidChecklistMetadataException
     );
   });
 });
 
-describe('hdf_profile_with_invalid_metadata', () => {
-  it('Throws InvalidChecklistFormatException when trying to convert to checklist with invalid metadata', () => {
-    // ensures that checklist metadata is being validated
-    const fileContents = JSON.parse(
-      fs.readFileSync(
-        'sample_jsons/checklist_mapper/sample_input_report/invalid_metadata.json',
-        {encoding: 'utf-8'}
-      )
-    );
-    expect(() => new ChecklistResults(fileContents)).toThrowError(
-      InvalidChecklistMetadataException
+describe('checklist_mapper_severity_mapping', () => {
+  it('Maps control V-61867 to correct severity category', () => {
+    const hdfData = loadJsonFile(
+      'sample_jsons/attestations/triple_overlay_profile_sample.json'
     );
+    const mapper = new ChecklistResults(hdfData);
+    const jsonixData = mapper.getJsonix();
+    const severity = extractSeverity(jsonixData, 2, 1);
+    const status = extractStatus(jsonixData, 2);
+    expect(severity).toBe('medium');
+    expect(status).toBe('Not_Applicable');
   });
 });
+
+/**
+ * Load and parse the file.
+ * @param filePath Path to the file.
+ * @returns Parsed data.
+ */
+function loadJsonFile(filePath: string): any {
```

Comment:
All previous calls to load the JsonFile indicates that the return type was a ChecklistResults, however we are not returning a ChecklistResults but the json. It may be the same, but wanted to mention it.

Reply:
I see what you mean here. I just broke it into two line in each test so that this function is used to load the file that will be input and then the ChecklistResults will be used with the data to create the mapper in a second line to make it readable.